dandi/dandiapi.py

```diff
@@ -397,7 +397,7 @@ def get_page(pageno: int) -> list:
 
 
 class DandiAPIClient(RESTFullAPIClient):
-    """A client for interacting with a Dandi Archive server"""
+    """A client for interacting with a DANDI server"""
```

Comment:
Just for completeness
```suggestion
    """A client for interacting with a DANDI API server"""
```